.github/workflows/release.yml

```diff
@@ -5,6 +5,18 @@ on:
     types: created
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '10.x'
+        registry-url: 'https://registry.npmjs.org'
+    - run: npm install
```

Comment:
I would use `npm ci` instead to make sure that what's published is exactly what's in the package-lock